### DISCUSSION TITLE
the-problem-of-data-grids-in-your-web-application/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#31] [Blog comments]
# the-problem-of-data-grids-in-your-web-application/

https://blogs.taiga.nl/martijn/posts/the-problem-of-data-grids-in-your-web-application/

**Comment by disqus-comments-migrator** [comment]
<em>René van den Berg commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at juni 16 2009, 04:21</em>

---

<p>Another solution is to use a RIA like Adobe's flex. Their grid is almost excel like.</p><p>René</p>

**Comment by disqus-comments-migrator** [comment]
<em>Martijn Laarmam commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at juli 29 2009, 03:57</em>

---

<p>Depends on your definition of "excell like".</p><p>Sortable/column dragable/Paginated datagrids dont have to be a pain at all with the right RIA framework (javascript/flash/silverlight).</p><p>If they want editable data grids with various controls in the cells ExtJS (amongst others) is a really good choice.</p><p>If they need excell's reporting/formula functionality there needs might just entail a PivotGrid with precompiled OLAP data, devexpress's pivotgrid goes even further and allows for a great deal of customization of the data on the client side.</p><p>If you render a data grid where you can't do anything with the headers then yeah i wholeheartedly agree that a nicely formated list will feel way more finished to a client.<br>If they want a real time editable pivotgrid then they really want to use excell in the browser.</p>
